packages/fast-components-class-name-contracts-base/src/select/index.ts

```diff
@@ -27,6 +27,11 @@ export interface SelectClassNameContract {
      */
     select__multiSelectable?: string;
 
+    /**
+     * select menu positioning wrapper
+     */
+    select__menuPositioningWrapper?: string;
```

Comment:
instead of wrapper, can you change the word to "region"?

Reply:
yep